homework-1/cash_register.py

```diff
@@ -0,0 +1,78 @@
+"""
+
+TASK 1
+
+Write a class to represent a Cash Register.
+You class should keep the state of price total and purchased items
+
+Below is a starter code:
+------------------------
+1. you can add new variables and function if you want to
+2. you will NEED to add accepted method parameters where required.
+For example, method add_item probably accepts some kind of an item?..
+3. You will need to write some examples of how your code works.
+
+"""
+
+
+class CashRegister:
+
+    def __init__(self):
+        self.total_items = []
+
+    def add_item(self, new_item, new_price):
+        self.total_items.append({"Name": new_item, "Price": new_price})
+        print("Item added successfully")
+
+    def remove_item(self, item_to_remove):
+        for item in self.total_items:
+            if item['Name'] == item_to_remove:
+                self.total_items.remove(item)
+                print(f"Item has been removed from cart")
+
+    def get_total(self):
+        total_price = 0
```

Comment:
You can use the `sum([x['Price'] for x in self.total_items])` here to calculate total price